cafebabel/templates/articles/detail.html

```diff
@@ -11,7 +11,17 @@
     {% endif %}
     <h1>{{ article.title }}</h1>
 
-    <p><time>Published on {{ article.creation_date.date() }}</time> — by {{ article.author }}<br>Reading time: {{ article.body|reading_time }} min</p>
+    <p>
+      Published on <time>{{ article.creation_date.date() }}</time>
```

Comment:
I think the W3C had writtent the `pubdate` attribute just for Cafebabel: https://www.w3.org/wiki/Html/Elements/time#HTML_Attributes
Let's consider ;)

Reply:
ok, done.